### PR TITLE
exp: updated metrics/exp/doc.go

### PR DESCRIPTION
**Pull request by ehnuje**
## Proposed changes

- Updated metrics/exp/doc.go
- Just make the format of doc.go aligned with other doc.go ("package XXX ...")

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [x] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: metrics/exp/doc.go [hunk 1]
```diff
@@ -14,6 +14,6 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
 
-// Hook go-metrics into expvar
-// on any /debug/metrics request, load all vars from the registry into expvar, and execute regular expvar handler
+// package exp hooks go-metrics into expvar on any /debug/metrics request,
+// load all vars from the registry into expvar, and execute regular expvar handler.
```
Comment (aidan-kwon) [inline metrics/exp/doc.go]:
```suggestion
// loads all vars from the registry into expvar, and executes regular expvar handler.
```

File: metrics/exp/doc.go [hunk 2]
```diff
@@ -14,6 +14,6 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
 
-// Hook go-metrics into expvar
-// on any /debug/metrics request, load all vars from the registry into expvar, and execute regular expvar handler
+// package exp hooks go-metrics into expvar on any /debug/metrics request,
```
Comment (aidan-kwon) [inline metrics/exp/doc.go]: What is /debug/metrics request?

Comment (ehnuje) [reply metrics/exp/doc.go]: @aidan-kwon I think `debug/metrics` should be replaced by `metrics`

**Comment by ehnuje** [comment]
@nogang @andybclee Could you please review this change?

File: metrics/exp/doc.go [hunk 3]
```diff
@@ -14,6 +14,6 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
 
-// Hook go-metrics into expvar
-// on any /debug/metrics request, load all vars from the registry into expvar, and execute regular expvar handler
+// package exp hooks go-metrics into expvar on any metrics request,
```
Comment (nogang) [inline metrics/exp/doc.go]: Shouldn't the first letter be capitalized?

Comment (ehnuje) [reply metrics/exp/doc.go]: Right! I will update this.

**Pull request merged**